e2e-test/go.mod

```diff
@@ -1,169 +1,134 @@
+	google.golang.org/protobuf v1.28.1 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiserver v0.24.5 // indirect
-	k8s.io/component-helpers v0.24.5 // indirect
-	k8s.io/kube-openapi v0.0.0-20220627174259-011e075b9cb8 // indirect
-	k8s.io/kubectl v0.24.2 // indirect
-	k8s.io/kubelet v0.0.0 // indirect
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.30 // indirect
-	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
-	sigs.k8s.io/kustomize/api v0.11.4 // indirect
-	sigs.k8s.io/kustomize/kyaml v0.13.6 // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
+	k8s.io/apiserver v0.26.1 // indirect
+	k8s.io/component-helpers v0.26.1 // indirect
+	k8s.io/kube-openapi v0.0.0-20221012153701-172d655c2280 // indirect
+	k8s.io/kubectl v0.26.1 // indirect
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.35 // indirect
+	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
+	sigs.k8s.io/kustomize/api v0.12.1 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.13.9 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
 	github.com/chaos-mesh/chaos-mesh => ../
 	github.com/chaos-mesh/chaos-mesh/api => ../api
-	k8s.io/api => k8s.io/api v0.24.5
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.24.5
-	k8s.io/apimachinery => k8s.io/apimachinery v0.24.5
-	k8s.io/apiserver => k8s.io/apiserver v0.24.5
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.24.5
-	k8s.io/client-go => k8s.io/client-go v0.24.5
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.24.5
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.24.5
-	k8s.io/code-generator => k8s.io/code-generator v0.24.5
-	k8s.io/component-base => k8s.io/component-base v0.24.5
-	k8s.io/component-helpers => k8s.io/component-helpers v0.24.5
-	k8s.io/controller-manager => k8s.io/controller-manager v0.24.5
-	k8s.io/cri-api => k8s.io/cri-api v0.25.0-alpha.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.24.5
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.24.5
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.24.5
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.24.5
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.24.5
-	k8s.io/kubectl => k8s.io/kubectl v0.24.5
-	k8s.io/kubelet => k8s.io/kubelet v0.24.5
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.24.5
-	k8s.io/metrics => k8s.io/metrics v0.24.5
-	k8s.io/mount-utils => k8s.io/mount-utils v0.24.5
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.24.5
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.24.5
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.24.5
-	k8s.io/sample-controller => k8s.io/sample-controller v0.24.5
-	vbom.ml/util => github.com/fvbommel/util v0.0.2
```

Comment:
it seems the issue about e2e is caused by these changes.

The e2e test part import the kubernetes e2e test framework, we need manually pin the kubernetes version.

We have a script to do that: `hack/update-kubernetes-library.sh`, that might be trick.

To fix that, I would push commits to this PR directly. Wish you do not mind this.

Reply:
Sure go ahead 😊

Reply:
I thought I would jump in to see if there is potentially a misunderstanding here.

@STRRL are you suggesting that @fasmat run

```bash
# The command will adjust ./go.mod and ./go.sum in the base directory of the repo
./hack/update-kubernetes-library.sh v1.26.1
```

and that he should commit and push those changes to this PR?

Are there any additional changes that need to be made to Ginkgo versioning for this PR to be ready? Will this require code changes?

Reply:
I re-added the replacements to the `go.mod` file and updated `ginkgo` to `v2.8.0`.

Reply:
> I thought I would jump in to see if there is potentially a misunderstanding here.
> 
> @STRRL are you suggesting that @fasmat run
> 
> ```shell
> # The command will adjust ./go.mod and ./go.sum in the base directory of the repo
> ./hack/update-kubernetes-library.sh v1.26.1
> ```
> 
> and that he should commit and push those changes to this PR?
> 
> Are there any additional changes that need to be made to Ginkgo versioning for this PR to be ready? Will this require code changes?

Sorry for the late response.